torchvision/transforms/functional_tensor.py

```diff
@@ -122,7 +122,14 @@ def hflip(img: Tensor) -> Tensor:
 def crop(img: Tensor, top: int, left: int, height: int, width: int) -> Tensor:
     _assert_image_tensor(img)
 
-    return img[..., top:top + height, left:left + width]
+    w, h = _get_image_size(img)
+    right = left + width
+    bottom = top + height
+
+    if left < 0 or top < 0 or right > w or bottom < h:
```

Comment:
Is this condition for `bottom` correct? I would have imagined that it should be similar than the one for `right` (which uses a `>` instead of a `<`).

Reply:
I missed it.
Fixed this.
Thank you!